contents/blog/is-google-analytics-illegal-microsite.md

```diff
@@ -0,0 +1,142 @@
+---
+date: 2022-03-01
+title: "Google Analytics was deemed illegal in some EU countries. We built a microsite in 48 hours to capitalize on the news."
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/is-ga-illegal-microsite.png
+author: ["cory-watilo"]
+categories: ["Inside PostHog"]
+---
+
+On January 14, 2022, the world learned that Austria declared Google Analytics to be illegal (in its default form), since it transmits IP addresses to the United States. Both Austria and The Netherlands both announced the news within days of each other.
+
+Given that PostHog is an open source, self-hostable analytics tool that eliminates the need to transmit data to *any* third party, this was obviously a perfect opportunity for us to capitalize on the news.
+
+Within five days, we launched [IsGoogleAnalyticsIllegal.com](https://isgoogleanalyticsillegal.com) and shared it with the Hacker News community. Here’s the story of how we built it in two days, how it was received by the HN community, and our missteps along the way.
+
+## A timeline
+
+### It all started when our CTO shared the news on Slack
+
+![The Netherlends also likely to declare Google Analytics illegal](../images/blog/is-ga-illegal-microsite/illegal-in-austria.png)
+
+### Then our VP of Ops came up with an idea to launch a microsite
+
+![Is this idea funny or just really dumb?](../images/blog/is-ga-illegal-microsite/is-this-funny-or-just-really-dumb.png)
+
+### Obviously this felt like a fun idea, so I put together a quick joke of a wireframe
+
+![Is GA illegal?](../images/blog/is-ga-illegal-microsite/is-it-illegal-probably.png)
+
+### Soon we had a gimmicky domain name and the idea was born
+
+![Greatest domain name ever](../images/blog/is-ga-illegal-microsite/is-google-analytics-illegal.png)
+
+### I remember being on a flight at the time. What better way to spend a couple hours in coach than to whip up a mock?
+
+![1 hour mockup](../images/blog/is-ga-illegal-microsite/mockup.png)
+
+## How we positioned the site
+
+We planned to launch this site on Hacker News, a site known for having an audience that really doesn’t like being sold to. If it doesn’t come across as authentic, be prepared to be torn to shreds.
```

Comment:
I would reframe this a bit as 'We decided to share the site on Hacker News'. Otherwise it sounds like we built a site specifically for the purposes of sharing on Hacker News, which definitely wasn't the plan - generally we make cool stuff first, and then think about where to share it. In this case, HN seemed like the obvious channel.